libsupport/include/katana/ArrowInterchange.h

```diff
@@ -279,12 +279,22 @@ TableBuilder::AddColumn(const ColumnOptions& options) {
 ////////////////////////////////////////////
 // Arrow utilities
 
-KATANA_EXPORT std::shared_ptr<arrow::Array> Unchunk(
+KATANA_EXPORT Result<std::shared_ptr<arrow::Array>> Unchunk(
```

Comment:
perhaps it is time to add some documentation to these functions?

Reply:
Sure, fair enough. Though I'm looking to kaizen this PR with a better set of functions, some documentation won't hurt.